app/models/lookup_value.rb

```diff
@@ -45,6 +45,7 @@ def name
   end
 
   def value_before_type_cast
+    return read_attribute(:value) if errors.present? && errors[:value].present?
```

Comment:
ok. errors is ActiveModel::Errors
it does respond to #present? with false when no errors, but I'll remove it :smile: